### DISCUSSION TITLE
An error occured<title>

### DISCUSSION THREAD

**Discussion by Shaurya25071997** [community/community#124711] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My githu bis showing an error occured while eeing jupyter notebooks

**Comment by Akshatapro** [comment]
Even I am having same issue

**Comment by mph4** [comment]
Hi Everyone - very sorry for the inconvenience here. We've fixed the problem and are also implementing improvements that we've learned from this to help prevent it (and better respond to any future issues) going forward.
